lib/image_picker_web.dart

```diff
@@ -1,15 +1,27 @@
 library image_picker_web;
 
 import 'dart:async';
-import 'dart:html' as html;
+import 'dart:js_interop';
+import 'dart:typed_data';
 
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:image_picker_web/src/models/media_info.dart';
+// import 'dart:html' as html;
+import 'package:web/web.dart' as web;
 
 export 'src/models/media_info.dart';
 
+/// Adds a `toList` method to [web.FileList] objects.
+extension WebFileListToDartList on web.FileList {
+  /// Converts a [web.FileList] into a [List] of [web.File].
+  ///
+  /// This method makes a copy.
+  List<web.File> get toList =>
+      <web.File>[for (int i = 0; i < length; i++) item(i)!];
```

Comment:
Let's keep an API cohesive with what already exists, prefer having a method instead of a getter:
```dart
List<web.File> toList() => [for (int i = 0; i < length; i++) item(i)!];
```